src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -608,6 +609,42 @@ public void cannotRenewWhenLoanPolicyDoesNotExist()
       "Loan policy %s could not be found, please check circulation rules", unknownLoanPolicyId)));
   }
 
+  @Test
+  public void canRenewLoanWithAnotherLoanPolicyName()
```

Comment:
Why is there a dedicated test for this?

Is there another test that already demonstrates the renewal process with a different policy?